core/src/main/java/org/elasticsearch/action/admin/cluster/node/tasks/list/ListTasksRequest.java

```diff
@@ -47,15 +48,31 @@ public ListTasksRequest detailed(boolean detailed) {
         return this;
     }
 
+    /**
+     * Should this request wait for all found tasks to complete?
+     */
+    public boolean shouldWaitForCompletion() {
+        return waitForCompletion;
+    }
+
+    /**
+     * Should this request wait for all found tasks to complete?
+     */
+    public void setWaitForCompletion(boolean waitForCompletion) {
```

Comment:
I've been asked in the past to make getters and setters for new stuff on requests. Particularly when doing reindex. I really don't care either way. I used `shouldWaitForCompletion` rather than `getWaitForCompletion` because  I thought that was more readable but I can make it whatever you think is more right.